src/android/patches/makeStringsPatch.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function makeStringsPatch(params) {
+  const patch = Object.keys(params).map(key =>
+    `    <string moduleConfig="true" name="${key}">${params[key]}</string>`
```

Comment:
> Meh, we can do it, but I think it's +1 dependency

It's not a +1 dependency since it's already in a tree. Just to recall what you said for lodash.

> What's the problem with default strings.xml?

Did I say there's any problem with that? To me it makes more sense to have a separate file that we fully manage instead of adding custom extraneous attribute `moduleConfig=""` that does not tell nothing.